testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1054,6 +1054,10 @@
   parent_channel = PARENT_CHANNEL_LABEL_TO_SYNC_BY_BASE_CHANNEL[product][base_channel]
   get_target('server').wait_while_process_running('mgr-create-bootstrap-repo')
 
+  $stdout.puts "base_channel: #{base_channel}"
+  $stdout.puts "channel: #{channel}"
+  $stdout.puts "parent_channel: #{parent_channel}"
+
```

Comment:
Don't we want to use `log` function or something alike?

Or did you just forget to remove these tests?

Reply:
From what I can see, we use both a lot. So I am also fine with changing that to `log " "`.

Reply:
I think it's Oscar who pushes for using log(). I am fine with that policy.

Reply:
Ok, then I am switching, too.

Reply:
Fixed in new commit.